.github/workflows/bench-all-runtimes.yml

```diff
@@ -0,0 +1,165 @@
+name: Bench all runtimes
+
+on:
+  # schedule:
+    # - cron: '0 1 * * 0' # weekly on Sunday night 01:00 UTC
```

Comment:
Disabled currently?

Reply:
yes, until we try it out few times manually, and adjust